packages/extension-chakra-storefront/src/theme/components/project/swatch-group.js

```diff
@@ -29,87 +32,108 @@ export default {
                 opacity: 1
             }
         }
-    }),
+    },
     variants: {
-        circle: (props) => ({
```

Comment:
FYI: The only modifications are 
1. Functions are removed to use plain key-value pairs
2. props of the functions is replaced with state modifiers (_hover, _selected, etc.) which is the recommended approach. 
There are no changes to the actual styling